src/Login/components/LoginComponent.jsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable jsx-a11y/anchor-has-content */
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/no-unescaped-entities */
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable react/jsx-closing-tag-location */
+/* eslint-disable react/jsx-indent */
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../LoginStyles/login.scss';
+
+
+const Loader = require('react-loader');
+
+const Login = (
+  {
+    handleSubmit,
+    onChange,
+    email,
+    password,
+    errors,
+    isLoggingIn,
+    loginFailed,
+    isLoading,
+  },
+) => (
+
+  <div className="Login-container">
+    <div className="heading">
+      <h1>Login</h1>
+    </div>
+    <div className="myform">
+      <form>
+        {
+          loginFailed
+            ? (
+          <div className="errors">
+            Please provide valid credentials
+            {loginFailed}
+            </div>
+            )
+            : (<span />)
+          }
+        <div className="form-input" id="errormessage">
+          <input type="text" id="userEmail" name="email" placeholder="Email" value={email} onChange={onChange} required />
+          {
+           errors.invalid_email
+             ? (
+               <div className="errors">
+               please provide valid email
+                 {errors.userEmail }
+               </div>
+             )
+             : (<span />)
+          }
+          {
+              errors.userEmail && (<div className="errors">{errors.userEmail}</div>)
+          }
+        </div>
+        <br />
+        <div className="form-input" id="errormessage">
+          <input type="password" value={password} placeholder="Password" name="password" onChange={onChange} />
+          {
+              errors.userPassword && (<div className="errors">{errors.userPassword}</div>)
+            }
+        </div>
+        <p className="mypara">Forgot Password?</p>
+        <br />
+        <Loader loaded={!isLoading}>
+        <div className="form-input">
+          <button type="button" onClick={handleSubmit} disabled={isLoggingIn}>
+            { isLoggingIn ? (<span>loading...</span>) : (<span>Login</span>)}
+          </button>
+        </div>
+        <p className="mypara2">
+          Or signup with
+          <a href="/" className="fa fa-facebook" />
+          <a href="/" className="fa fa-twitter" />
+          <a href="/" className="fa fa-google" />
+        </p>
+        </Loader>
+
+      </form>
+    </div>
+
+  </div>
+);
+
+Login.propTypes = {
+  email: PropTypes.string,
```

Comment:
If this prop is not required then there has to be a default. Think about proptypes as the minimum data your component needs to function properly. Some are required and others are not. Those that're not required should provide a fallback value for the prop e.g. if prop X that's a number was not required by your component we would validate props like so;
```
YourComponent.propTypes = {
X: PropTypes.number,
}
```
Then we provide a default because since it's not required the user may not pass it in, like so;
```
YourComponent.defaultProps = {
X: 0,
}
```

Reply:
@AGMETEOR  kindly note l have worked on this thank you